ext/opcache/Optimizer/zend_func_info.c

```diff
@@ -951,7 +951,7 @@ static const func_info_t func_infos[] = {
 	F1("mysqli_fetch_all",						MAY_BE_NULL | MAY_BE_FALSE | MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_ANY | MAY_BE_ARRAY_OF_ANY),
 	F1("mysqli_fetch_object",					MAY_BE_NULL | MAY_BE_FALSE | MAY_BE_OBJECT),
 	F0("mysqli_free_result",					MAY_BE_NULL),
-	F0("mysqli_affected_rows",					MAY_BE_NULL | MAY_BE_LONG),
+	F0("mysqli_affected_rows",					MAY_BE_NULL | MAY_BE_STRING | MAY_BE_LONG),
```

Comment:
These should all be changed to the `F1` macro now. While at it, can you please also swap the order to `MAY_BE_LONG | MAY_BE_STRING`?

Reply:
Done